ci/dhis2-verify-node.yml

```diff
@@ -61,5 +61,4 @@ jobs:
             - uses: c-hive/gha-yarn-cache@v1
             - run: yarn install --frozen-lockfile
 
-            - name: Publish to NPM
```

Comment:
(minor) any reason to remove the name here?

Reply:
I found it a lot easier to find what I wanted when I saw the command as the "name" instead of a customized name. Sometimes the command changes and the name is not updated to reflect it. We could make it more generic, but I think it's moot because I would like to replace L64-65 with https://github.com/dhis2/action-semantic-release/